src/controller/java/src/chip/devicecontroller/ICDClientInfo.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *   Copyright (c) 2020-2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package chip.devicecontroller;
+
+/** Class for holding ICD Client information. */
+public class ICDClientInfo {
+  private final long nodeId;
```

Comment:
naming: peerNodeId?
nodeId is scoped with fabric
so you may create a new class which is equivalent with ScopedNodeId ?

Reply:
The name was changed to peerNodeId. And, in the case of ChipDeviceController, since an object is created for each Fabric, I don't think it needs to be configured with ScopedNodeId. Do you have any other opinions?

Reply:
i see, thanks for clarification.